src/main.py

```diff
@@ -135,7 +134,7 @@ async def post_pv_actual(
 
     logger.debug(f"Adding {len(generation_values_df)} generation values")
 
-    _ = insert_generation_values(session=session, generation_values_df=generation_values_df)
+    insert_generation_values(session, generation_values_df)
```

Comment:
i like hvaing the `session=session`, helps keep it more robust for the future. Perhaps there's two different school of thought here

Reply:
Honestly I changed it because first I moved the "write" helper functions in here and them move them back and locally it was clear enough. Also I think those should be refactored away at some point, so I didn't spend much time cleaning them.

Generally if arguments are clear and couldn't be confusing even if we tried, I think positional arguments are fine. Since the first argument here will always be called `session` and clearly be a session, in any context, and the second argument will always look like a dataframe, I thought it was fine.

But I don't mind changing it back.